src/shared/containers/TableViewerContainer.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import { Modal } from 'antd';
+import { useNexusContext } from '@bbp/react-nexus';
+import * as csvParser from 'csv-string';
+
+import TableViewer from '../components/TableViewer';
+import useNotification from '../hooks/useNotification';
+
+const TableViewerContainer: React.FC<{
+  resourceUrl: string;
+  name: string;
+  orgLabel: string;
+  projectLabel: string;
+  onClickClose: () => void;
+}> = ({ resourceUrl, orgLabel, projectLabel, name, onClickClose }) => {
+  const nexus = useNexusContext();
+  const [tableData, setTableData] = React.useState<any>();
+  const notification = useNotification();
+
+  React.useEffect(() => {
+    nexus.File.get(orgLabel, projectLabel, encodeURIComponent(resourceUrl), {
```

Comment:
Can we use async-await here? You will have to add a method with the async keyword and then call it inside the useEffect. (this is nit pick, you can skip this if you don't feel like doing it :) .